src/contracts/ValueDecoder.sol

```diff
@@ -0,0 +1,10 @@
+//SPDX-License-Identifier: Unlicensed
+pragma solidity 0.6.8;
+
+library ValueDecoder {
+  function toUint(bytes memory _bytes) internal pure returns (uint256 value) {
```

Comment:
Will we have ValueEncoder?

uint256 a;
a.toBytes()

a.toBytes().toUInt()

Reply:
why we need encoder? atm I see no reason to have it here.